concourse/scripts/val_from_yaml.rb

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
```

Comment:
I assume this script has been grabbed from somewhere online, would be nice to have the link in the commit message for future reference.